frontend/README.md

```diff
@@ -127,6 +127,24 @@ A subset of tests can be ran by passing a pattern to the script. For example, to
 npm run test-watch -- pages
 ```
 
+## 🚦 End-to-end (E2E) testing
+
+[Playwright](https://playwright.dev/) is a framework for web testing and its test runner is called [Playwright Test](https://playwright.dev/docs/api/class-test), which can be used to run E2E or integration tests across chromium, firefox, and webkit browsers.
+
+E2E test filenames end with `.spec.ts` and are found in the `tests/e2e` directory.
+
+To run E2E tests via CLI:
+
+- `npm run test:e2e` — Runs all E2E tests using the playwright config found at `tests/playwright.config.ts`
+- `npm run test:e2e:ui` — Run specific or all E2E tests using Playwright's [UI mode](https://playwright.dev/docs/test-ui-mode), which is useful for debugging full traces of each test
```

Comment:
```suggestion
- `npx playwright install` — Installs required browsers
- `npm run test:e2e` — Runs all E2E tests using the playwright config found at `tests/playwright.config.ts`
- `npm run test:e2e:ui` — Run specific or all E2E tests using Playwright's [UI mode](https://playwright.dev/docs/test-ui-mode), which is useful for debugging full traces of each test
```

The tests wouldn't work for me until I installed Playwright independently of the `@playwright/test` dev dependency.

Reply:
@acouch I didn't immediately realize that your comment was a suggestion, so I pushed up another commit with similar changes.